.hound.yml

```diff
@@ -4,7 +4,3 @@
 jshint:
   config_file: .jshintrc
   ignore_file: .jshintignore
-
-rubocop:
-  version: 1.22.1
-  config_file: .rubocop.yml
```

Comment:
Sorry, I forgot to mention that to disable rubocup, it needs to be explicitly be disabled because it [defaults to being enabled](http://help.houndci.com/en/articles/2137566-rubocop):
```
rubocop:
  enabled: false
```